INSTALL.md

```diff
@@ -94,12 +96,17 @@ to build a portable binary, add `PORTABLE=1` before your make commands, like thi
               sudo yum install libasan
 
     * Install zstandard:
+        * With [EPEL](https://fedoraproject.org/wiki/EPEL) or CentOS 8:
+
+              sudo yum install libzstd-devel
```

Comment:
I am under the impression that `dnf` is preferred on CentOS 8. https://centos.pkgs.org/8/elrepo-testing-x86_64/zstd-1.3.4-1.el8.elrepo.x86_64.rpm.html

Reply:
Tested on my CentOS 8 box with the following
```
dnf install libzstd-devel
```

Reply:
`yum` works as well with CentOS 8, and `dnf` is not mentioned elsewhere in this doc for CentOS/RHEL. Also it appears `yum` is symlinked to `dnf` on CentOS 8:

```
[root@6ae6dc102717 /]# ls -laF $(command -v yum)
lrwxrwxrwx 1 root root 5 Aug  4  2020 /usr/bin/yum -> dnf-3*
[root@6ae6dc102717 /]# ls -laF $(command -v dnf)
lrwxrwxrwx 1 root root 5 Aug  4  2020 /usr/bin/dnf -> dnf-3*
```

Reply:
I would suggest to stick with Yum as it is probably the most used and can is operable on all versions of CentOS.

Reply:
> Also it appears `yum` is symlinked to `dnf` on CentOS 8:

Not on one of my machines, but true on the other. So I think this assumption does not hold.
On the first CentOS 8 machine, I tried to use yum install, but I got error "yum not found".

FWIW, I think the doc should try to be up-to-date with what the OS recommends. dnf is going to replace yum, making this doc out-dated again.

Reply:
> I would suggest to stick with Yum as it is probably the most used and is operable on all versions of CentOS.

Unless the CentOS 8 in the above line is removed.

Reply:
Ok broke out CentOS 8 on its own here with `dnf`.

Reply:
[unrelated to this specific PR] Speaking of out-dated, our suggestion to install ZSTD 1.1.3 on non-CentOS 8 could be updated as well. I wonder if there's a URL for the latest release code.

Reply:
I don't know of any direct link to the latest release source on github, partly because zstd puts the release tag in the asset name. There are lots of examples on scripted solutions that use curl to query the github api and parse out the latest release tag using jq/sed/grep whatever, and then create a download link from that, e.g.: https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c

That could be an option, assuming the latest-in-time zstd is always what folks should use.